src/app/(pages)/groups/components/table/options/remove-membership-button/index.tsx

```diff
@@ -0,0 +1,38 @@
+"use client";
+import { useState } from "react";
+import ConfirmUnlinkUserModal from "./modal";
+import { ScimReference } from "@/models/scim";
+
+export type RemoveMembershipProps = {
+  userRef: ScimReference;
+  groupId: string;
+  groupName: string;
```

Comment:
Since you are passing both groupId and groupName, is it possibile to pass the entire `Group` object?
If so, we lower the chances to write a `groupId` and a `groupName` that are not related to each other.